src/main/resources/org/jenkinsci/plugins/workflow/steps/WriteFileStep/help-file.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Relative path of a file within the workspace.
```

Comment:
Similar to the Pushd documentation, do you want to want to make this match terminology by using `directory` here?  I think it makes sense to match but want to confirm /cc @StackScribe

Reply:
@kwhetstone, I thought directory in only used to specify a folder not a file.

Reply:
I'm not sure of the context here -- does this parameter specify the directory where the file will be written or the path (including he file name) of the file.  That would determine the proper wording here.

Reply:
@StackScribe, the path of the file.